hazelcast/src/main/java/com/hazelcast/memory/MemorySize.java

```diff
@@ -37,15 +39,14 @@ public MemorySize(long value, MemoryUnit unit) {
         if (value < 0) {
```

Comment:
Good point!

GitHub should have a 'resolved' feature for line comments, where no change is done eventually.